devtools/dump_devinfo.py

```diff
@@ -14,6 +14,8 @@
 import re
 import traceback
 from collections import defaultdict, namedtuple
+from os import listdir
+from os.path import isfile, join
```

Comment:
Better use pathlib and Path.rglob() for collecting the fixtures, which I think you are using these.

Reply:
Done